build/stf-run-ci/defaults/main.yml

```diff
@@ -26,7 +26,7 @@ __service_telemetry_snmptraps_trap_oid_prefix: "1.3.6.1.4.1.50495.15"
 __service_telemetry_snmptraps_trap_default_oid: "1.3.6.1.4.1.50495.15.1.2.1"
 __service_telemetry_snmptraps_trap_default_severity: ""
 __service_telemetry_logs_enabled: false
-__service_telemetry_observability_strategy: use_community
+__service_telemetry_observability_strategy: use_hybrid
```

Comment:
So stf-run-ci defaults to use_hybrid (keeps our existing test structure currently, but uses OBO for metrics storage vs community catalogsource for Prometheus), but then I noticed the default value in roles/servicetelemetry/defaults/main.yml is targeting use_redhat (for new installations).

I was initially a bit confused, but I get it now, and just wanted to write this down to verify.

New STF installations will have use_redhat enabled because that's what the default/main.yml will have, along with the implicit value set in the CR (which in turn also populates the OCP UI interface when you create an STF object from the console).

We will test via use_hybrid for now to keep our larger test coverage, but it will default to using OBO instead of community prometheus as the metrics data store.

I went through the code here, and it looks like use_hybrid is basically use_community (components wise) but OBO vs Prometheus Community.

There is the other PR #422 to narrow the ES scope, but if stf-run-ci is using use_hybrid, doesn't that mean we have all the necessary components to test events storage? I feel like I'm missing something here and I can't see it.

Reply:
Your explanations are all correct. 

As for PR #422 , yes, stf-run-ci does default to use_hybrid, but #422 asks Jenkins to deploy with use_redhat instead and limits the smoke test to only that coverage by default. On THIS PR, I noted that testing with use_hybrid and deploying with use_redhat by default seems like a bad idea. #422 is the solution to that problem, but comes at the cost of removing all ES related testing (ES testing should return when we add the supported forwarding model). I proposed it separately to try to keep this focused and prevent altering the smoketests alongside a large change.